tests/pl/homeassistant_HassGetCurrentDate.yaml

```diff
@@ -0,0 +1,20 @@
+language: pl
+tests:
+  - sentences:
+      - "jaki mamy dzień"
+      - "jaki dzisiaj mamy dzień"
+      - "jaki obecnie mamy dzień"
+      - "jaki mamy dzisiaj dzień"
+      - "jaki mamy obecnie dzień"
+      - "jaki dzisiaj jest dzień"
+      - "jaki obecnie jest dzień"
+      - "jaki jest dzisiaj dzień"
+      - "jaki jest obecnie dzień"
+      - "podaj mi datę"
+      - "podaj mi dzisiejszą datę"
+      - "podaj mi obcną datę"
+      - "podaj mi aktualną datę"
+      - "jaka jest dzisiejsza data"
+    intent:
+      name: HassGetCurrentDate
+    response: Jest wtorek, siedemnasty września dwutysięcznego trzynastego roku.
```

Comment:
We cannot use this placeholder because Polish voice synthesizers read the hours incorrectly, so I decided to transform it so that they are pronounced correctly. I added extra sentences..